src/kirby/components/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+declare var require: any;
+const style: any = require('sass-extract-loader!./checkbox.component.scss');
+
+@Component({
+  selector: 'kirby-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+})
+export class CheckboxComponent implements OnInit {
+  @Input() checked: boolean;
+  @Input() color: string;
+  @Input() shape: string;
+  @Output() checkedChange = new EventEmitter<boolean>();
+
+  classes: string[] = [];
+
+  get nativeColor(): string {
```

Comment:
Consider moving this behaviour to `ngOnChanges` and exposing through a dedicated property... the current implementation might be "costly", based on the implementation of `sass-extract-loader`.

Reply:
I might say Ihave no idea how to make a "proper" dedicated property to extract values of css variables of colors!